src/tests/ibus-share.c

```diff
@@ -4,10 +4,19 @@
 static void
 test_machine_id (void)
 {
+    DBusError error = DBUS_ERROR_INIT;
     const gchar *s1 = ibus_get_local_machine_id ();
-    gchar *s2 = dbus_get_local_machine_id ();
+    gchar *s2 = dbus_try_get_local_machine_id (&error);
 
-    g_assert_cmpstr (s1, ==, s2);
+    g_assert (s1);
+    if (!s2) {
+        g_warning ("Failed to get locale machine id: %s: %s",
```

Comment:
```suggestion
        g_warning ("Failed to get local machine id: %s: %s",
```

Locales are a translation/internationalization thing, and are not related to the machine ID of the "local machine" (the machine where you're running this test).

Reply:
Thank you. It was my typo.